codice/ddf
# DDF-4077 Remove Standard Search UI and CometD from the codebase

#### What does this PR do?
Removes Standard Search UI and CometD from the codebase. 
This reduces the distribution by ~95MB
#### Who is reviewing it? 
@peterhuffer @kcover 
#### Ask 2 committers to review/merge the PR and tag them here.

@andrewkfiedler 
@ricklarsen - Documentation
@shaundmorris
@vinamartin

#### How should this be tested?
Build and install DDF. Verify that DDF installs correctly and that there is no longer any standard search ui or cometD feature/bundle.

#### What are the relevant tickets?
[DDF-4077](https://codice.atlassian.net/browse/DDF-4077)

#### Checklist:
